components/vocabularies/hpo/annotations/src/test/java/org/phenotips/vocabulary/internal/hpoannotations/NegativePhenotypeForDiseaseAnnotationsExtensionTest.java

```diff
@@ -83,6 +105,17 @@
     @Mock
     private Vocabulary vocabulary;
 
+    @Mock
+    private XWiki xwiki;
+
+    @Mock
+    private XWikiDocument document;
+
+    @Mock
+    private BaseObject object;
+
+    private VocabularySourceRelocationService serv;
```

Comment:
...and similarly it is easier to understand whats going on if this is a `relocationServer` (or something similar), not a generic "serv"\"server"